.eslintrc.json

```diff
@@ -2,22 +2,33 @@
   "env": {
     "browser": true,
     "node": true,
-    "es2021": true,
+    "es2020": true,
     "jest": true
   },
   "extends": [
     "eslint:recommended",
     "plugin:react/recommended",
+    // "plugin:react/jsx-runtime",
```

Comment:
please dont keep comments for stuff that is not required. delete and push changes